web/themes/custom/vote_gov/templates/partial/register/registration-type--online.html.twig

```diff
@@ -4,8 +4,16 @@
   <h2>{{ 'How to register to vote in @state_name' | t({ '@state_name': state_name }) }}</h2>
     {# registration link, more info link #}
     {% if content.field_registration_link | render %}
+    
       {% set registration_link = content.field_override_registration_link | field_value | render | default( content.field_registration_link | field_value | render ) %}
+
       {% set registration_link_text = "Start your online registration" | t %}
+
+      {% if language != "en" and content.field-override-registration-link is empty %}
+          {% set in_english = "(in English)" | t %}
+          {% set registration_link_text = registration_link_text ~ in_english %}
```

Comment:
We need to add a space here 
`{% set registration_link_text = registration_link_text ~ ' ' ~ in_english %}`